build/cluster-operator-ansible/Dockerfile

```diff
@@ -0,0 +1,33 @@
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM openshift/origin-ansible:latest
```

Comment:
Probably best to build off v3.9 given what we were seeing with latest (v3.10) running against a v3.9 AMI breaking.

We probably need to switch to using this in the example cluster version in contrib/examples/cluster.yaml as well. We have one ansible image to define per cluster version, and should probably use it for every job.

This likely implies we're going to need to build an image for v3.9, v3.10, any supported major release. This isn't ideal but I don't see a way around it, we need to layer on top of stable openshift-ansible containers, we need to move faster than they do, but we need to move slower than openshift-ansible latest. (as it appears it's not backwards compat)

Reply:
Just v3.9 is fine for now, we can add v3.10 later when we start doing builds.

Reply:
@dgoodwin we actually pull the latest container, but then overwrite the /usr/share/ansible/openshift-ansible directory with a customizable checkout. So we're really just using origin-ansible:latest for the packages they build into it. We currently clone $CLUSTER_OP_ANSIBLE_REPO:$CLUSTER_OP_ANSIBLE_BRANCH on top of /usr/share/ansible/openshift-ansible. I'll set the default of $CLUSTER_OP_ANSIBLE_BRANCH to 3.9 instead of master.

Reply:
Works for me, thanks.

Reply:
latest commit i just pushed moves to to a default branch of release-3.9